rdbms/src/main/java/io/spine/server/storage/jdbc/query/AbstractQuery.java

```diff
@@ -128,16 +134,56 @@ protected PathBuilder<Object> aliasedPathOf(TableColumn column) {
         return aliasedPathBuilder.get(column.name());
     }
 
+    @SuppressWarnings("rawtypes")   /* The exact type of `Comparable` is not known here. */
     protected <T extends Comparable> ComparablePath<T>
     aliasedComparablePathOf(TableColumn column, Class<T> type) {
         return aliasedPathBuilder.getComparable(column.name(), type);
     }
 
+    @SuppressWarnings("rawtypes")   /* The exact type of `Comparable` is not known here. */
     protected OrderSpecifier<Comparable> orderBy(TableColumn column, Order order) {
         PathBuilder<Comparable> columnPath = pathBuilder.get(column.name(), Comparable.class);
         return new OrderSpecifier<>(order, columnPath);
     }
 
+    /**
+     * Appends the given query with the ordering and limit, if they are provided.
+     *
+     * <p>In case the ordering directive is not specified, the limit value is ignored.
+     * Otherwise, the limit is applied if and only if it is a positive number.
```

Comment:
Should we throw an exception if it's a negative number? Also, can it ever be `0` or does the `null` value handle that option? I see that the behaviour is the same in both cases.

Reply:
Addressed.